src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolutionResult.cs

```diff
@@ -476,8 +476,8 @@ private static ThreeState TryGetBestResult(ArrayBuilder<MemberResolutionResult<T
                     && firstSupported.Result.Kind == MemberResolutionKind.NoCorrespondingNamedParameter)
                 {
                     int badArg = firstSupported.Result.BadArgumentsOpt[0];
-                    IdentifierNameSyntax badName = arguments.Names[badArg];
-                    diagnostics.Add(ErrorCode.ERR_FunctionPointersCannotBeCalledWithNamedArguments, badName.Location);
+                    Location badName = arguments.Names[badArg].Value.Location;
```

Comment:
I get the change to not take an actual SyntaxNode, but just using a tuple seems kind of... leaky. I guess it comes down to a team preference of tuples vs dedicated structs.

Reply:
A dedicated struct just felt like overkill here.